packages/cli/test/integration/tags.api.test.ts

```diff
@@ -0,0 +1,38 @@
+import * as Db from '@/Db';
+import * as utils from './shared/utils/';
+import * as testDb from './shared/testDb';
+import type { SuperAgentTest } from 'supertest';
+
+let authOwnerAgent: SuperAgentTest;
+const testServer = utils.setupTestServer({ endpointGroups: ['tags'] });
+
+beforeAll(async () => {
+	const globalOwnerRole = await testDb.getGlobalOwnerRole();
+	const ownerShell = await testDb.createUserShell(globalOwnerRole);
+	authOwnerAgent = testServer.authAgentFor(ownerShell);
+});
+
+beforeEach(async () => {
+	await testDb.truncate(['Tag']);
+});
+
+describe('POST /tags', () => {
+	test('should create tag', async () => {
+		const resp = await authOwnerAgent.post('/tags').send({ name: 'test' });
+		expect(resp.statusCode).toBe(200);
+
+		const dbTag = await Db.collections.Tag.findBy({ name: 'test' });
+		expect(dbTag.length === 1);
+	});
+
+	test('should not create duplicate tag', async () => {
+		const newTag = Db.collections.Tag.create({ name: 'test' });
+		await Db.collections.Tag.save(newTag);
+
+		const resp = await authOwnerAgent.post('/tags').send({ name: 'test' });
+		expect(resp.status).toBe(500);
```

Comment:
maybe out of scope of this PR, but we should update this endpoint to return `409` or `422` in case the tag already exists. That way, if there is some other bug causing an uncaught exception, the status would be 500, and this test would fail.